lib/spaceship/portal/portal_client.rb

```diff
@@ -377,6 +377,14 @@ def download_provisioning_profile(profile_id, mac: false)
     end
 
     def delete_provisioning_profile!(profile_id, mac: false)
+      if csrf_tokens.count == 0
+        r = request(:post, "account/#{platform_slug(mac)}/profile/getProvisioningProfile.action", {
```

Comment:
Yes. 
I doubled checked it with provisioning_profile which make a call to different endpoint.
It simply because response to provisioning_profiles does not include the csrf_ts and csrf tokens.